src/app/common/hooks/account/use-account-names.ts

```diff
@@ -16,5 +16,5 @@ export function useCurrentAccountDisplayName() {
 
 export function useAccountDisplayName(account: SoftwareWalletAccountWithAddress) {
   const names = useGetAccountNamesByAddressQuery(account.address);
-  return useMemo(() => names[0] ?? getAccountDisplayName(account), [account, names]);
+  return useMemo(() => names[0] ?? getAccountDisplayName(account));
```

Comment:
Is there a reason you removed the dependency array here?

Reply:
Hm nope, not sure why I did that in hindsight. I've pushed a new commit. 🙏